napari/_vispy/layers/image.py

```diff
@@ -174,6 +186,39 @@ def _on_attenuation_change(self) -> None:
         if isinstance(self.node, VolumeNode):
             self.node.attenuation = self.layer.attenuation
 
+    def change_render_quality(self, quality_change: RenderQualityChange):
+        """
+        Change the render quality of the vispy nodes.
+
+        This changes the step size in the shader when in 3D rendering mode.
+        In 2D rendering mode, this just returns.
+
+        Parameters
+        ----------
+        quality_change: RenderQualityChange
+            how much to increase or decrease the rendering quality of the layer.
+        """
+        if not isinstance(self.node, VolumeNode):
+            return
+        if quality_change == RenderQualityChange.DECREASE:
+            new_step_size = min(
+                self.node.relative_step_size * 4, self.max_step_size
+            )
+        elif quality_change == RenderQualityChange.INCREASE:
+            new_step_size = max(
+                self.node.relative_step_size / 2, self.min_step_size
+            )
+        elif quality_change == RenderQualityChange.MIN:
+            new_step_size = self.max_step_size
+        elif quality_change == RenderQualityChange.MAX:
+            new_step_size = self.min_step_size
+        self.node.relative_step_size = new_step_size
+
+        # update the minimum step size based on current performance
+        # we don't go below 1 because we want to be able to render accurately
+        # to the level of a single data pixel
+        self.min_step_size = np.clip(new_step_size / 4, 0.05, 1)
```

Comment:
```suggestion
        self.min_step_size = np.clip(new_step_size / 4, 0.1, 1)
```
Same as above, this needs to be min 0.1